djangocms_text_ckeditor/tests/frontend/integration/smoke.js

```diff
@@ -171,11 +173,12 @@ casper.test.begin('CKEditor loads correctly in collapsed fieldset', function (te
             this.click('input[type=submit]');
         })
         .waitForSelector('.success', function () {
-            test.assertSelectorHasText('.success', 'The pizza "Pizza object" was added successfully.', 'Pizza added');
+            test.assertSelectorHasText('.success', 'was added successfully.', 'Pizza added');
+            test.assertSelectorHasText('.success', 'Pizza object (2)', 'Pizza added');
         })
         .thenOpen(globals.baseUrl + 'admin/test_app/pizza/')
         .then(function () {
-            this.clickLabel('Pizza object');
+            this.clickLabel('Pizza object (2)');
```

Comment:
this needs to be clickable regardless of it having a number in the name or not, which is probably better to do with a combination of http://docs.casperjs.org/en/latest/modules/casper.html#click and an xPath selector, e.g.

```suggestion
this.click(xPath('//[contains(text(), 'Pizza object')]'))
```

where `xPath` is 
```
var casperjs = require('casper');
var xPath = casperjs.selectXPath;
```